Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/Getting-Started.md

```diff
@@ -62,40 +65,69 @@ Follow the steps [here](https://learn.microsoft.com/virtualization/windowscontai
 1. If you are running these commands on your domain controller that is a Windows Server machine, you may have trouble
 installing the [Kubernetes CLI (kubectl)](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/) and [kubelogin](https://github.com/Azure/kubelogin). The error would mention "Install-AzAksKubectl". If so, you will need to manually install both following the links above.
 
-## How to Validate Your GMSA Integration
+## How to Validate Your GMSA Integration before Deployment
+
+1. To validate that your cluster is successfully retrieving your GMSA, go into your domain controller local server menu, go to Tools and select Event Viewer. Look under ActiveDirectory events. Look at the contents of the most recent events for a message that says "A caller successfully fetched the password of a group managed service account." The IP address of the caller should match one of your AKS cluster IPs.
+
+## Import Simple Application Container Image
+
+To build the application container image, you will need to import the Windows Server 2019 LTSC image to your Azure Container Registry that was deployed as apart of the reference architecture. Once this image is imported, you will reference the version from your ACR in the workload manifest rather than the public image from Microsoft Container Registry. 
+
+```PowerShell
+# enter the name of your ACR below
+$SPOKERG=<resource group name for spoke>
+$ACRNAME=$(az acr show --name <ACR NAME> --resource-group $SPOKERG --query "name" --output tsv)
+```
+
+Log into ACR
+
+```PowerShell 
+az acr login -n $ACRNAME
+```
+
+Import the Windows Server 2019 LTSC image into the container registry. Ensure you are logged into the Azure Container Registry, you should show a successful login from the command above.
```

Comment:
Don't even need to log in when doing an `az acr import` as that uses your RBAC access :)

```suggestion
Import the Windows Server 2019 LTSC image into your container registry.
```